### PR TITLE
Max zoom config

### LINKED ISSUE #2492: The max zoom on (point) search should be configurable
https://github.com/camptocamp/ngeo/blob/2.1/contribs/gmf/src/directives/search.js#L841

### PR DESCRIPTION
**Pull request by sbrunner**
Fix #2492

### CODE REVIEW

File: contribs/gmf/src/directives/search.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ gmf.module.value('gmfSearchTemplateUrl',
  *      change the style of the feature on the map. Default is false.
  * @htmlAttribute {ngeox.SearchDirectiveListeners} gmf-search-listeners
  *      The listeners.
+ * @htmlAttribute {number} gmf-max-zoom The maximum zoom we will zoom on result, default is 16.
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/search.js]: Better coherence with name: `gmf-search-maxzoom`

File: contribs/gmf/src/directives/search.js [hunk 2]
```diff
@@ -135,7 +136,8 @@ gmf.searchDirective = function(gmfSearchTemplateUrl) {
       'clearbutton': '=gmfSearchClearbutton',
       'colorchooser': '=gmfSearchColorchooser',
       'coordinatesProjections': '=?gmfSearchCoordinatesprojections',
-      'additionalListeners': '=gmfSearchListeners'
+      'additionalListeners': '=gmfSearchListeners',
+      'maxZoom': '<gmfMaxZoom'
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/search.js]: With my previous comments:

`'maxZoom': <gmfSearchMaxzoom`

(/!\ watch out the uppercase)

File: contribs/gmf/src/directives/search.js [hunk 3]
```diff
@@ -274,6 +276,12 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
    */
   this.placeholder = '';
 
+  /**
+   * The maximum zoom we will zoom on result.
+   * @type {number}
+   */
```
Comment (fredj) [inline contribs/gmf/src/directives/search.js]: missing `@export`

File: contribs/gmf/src/directives/search.js [hunk 4]
```diff
@@ -274,6 +276,12 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
...
+  this.maxZoom = 16;
```
Comment (fredj) [inline contribs/gmf/src/directives/search.js]: Does it work? (default value)

Comment (ger-benjamin) [inline contribs/gmf/src/directives/search.js]: I think but not 100% sure... can you try @sbrunner ?

Comment (sbrunner) [inline contribs/gmf/src/directives/search.js]: Yes :-)

**Comment by sbrunner** [comment]
Updated :-)

**Pull request merged**